code/Gamemodes/Modes/FFA/FreeForAll.cs

```diff
@@ -47,6 +47,14 @@ internal override void Start()
 		base.Start();
 	}
 
+	internal override void IncrementGamesPlayedStat()
+	{
+		foreach ( var player in Players.Where( p => !p.Client.IsBot ) )
+		{
+			Sandbox.Services.Stats.Increment( player.Client, "ffa-games-played", 1 );

```

Comment:
Let's write a helper or util class for stats that codifies the name of the stat into an enum or const string.